opus/application/static_media/js/browse.js

```diff
@@ -1801,6 +1831,9 @@ var o_browse = {
         // list columns + values
         let html = "<dl>";
         $.each(opus.colLabels, function(index, columnLabel) {
+            if (opusId === "" || o_browse.galleryData[opusId] === undefined || o_browse.galleryData[opusId][index] === undefined) {
+                console.log("uh oh");
```

Comment:
Ummmmm....

Nothing wrong with doing a sanity change, but go ahead and use the new opus.logError function and make the description meaningful

Reply:
oops, lol.  that was just for me to debug... I meant to remove it.  Ah well...